### DISCUSSION TITLE
How to get Franka Panda Robot Arm seize objects tightly

### DISCUSSION THREAD

**Discussion by song-siqi** [google-deepmind/mujoco#2000] [Asking for Help]
Hi,

I'm a student and I'm trying to use MuJoCo for a project on robot manipulation. I'm looking for some help with the model of menagerie.

I tried to control the Panda hand to seize a cube or a rectangular box, resulting in **the fingers slipping from the object after I tried to move other joints to lift it up**.
I used `data.ctrl` to control the actuator of the Franka Panda hand fingers using python, only to find that **the fingers can be easily separated** by the box, which can be seen in the video.

I am looking for some hard-coding policy codebases for reference, or is there any parameters I can adjust to make the tendon between fingers tighter?

https://github.com/google-deepmind/mujoco/assets/129511833/6a916abe-3821-4a7e-9e63-163bb9a462c3

**Comment by Balint-H** [comment]
You can experiment with the xml, editing the hand actuator's force limits or gain. Then again, this behaviour may be accurate, and a control policy should take into account it should lift the object before moving it (so it is not in contact with the ground)
